WooCommerce/Classes/POS/Presentation/ItemRowView.swift

```diff
@@ -78,6 +78,15 @@ struct ItemRowView: View {
 }
 
 private extension ItemRowView {
+    var cardOutlineWidth: CGFloat {
```

Comment:
nit: could also use the same color as the row for dark mode (`posCartItemOutline` returning `posSecondaryBackground` in dark mode), so that we don't need to return different outline widths?

Reply:
Could do... but I think this way makes it clearer in the code that there shouldn't be an outline when we're in dark mode.